build/templates/grpc_session_options.py.mako

```diff
@@ -1,7 +1,17 @@
-# -*- coding: utf-8 -*-
+${template_parameters['encoding_tag']}
+# This file was generated
+<%
+    config = template_parameters['metadata'].config
+%>\
+
 from enum import IntEnum
 
 
+# This constant specifies the gRPC package and service used by this API.
+# MeasurementLink customers should pass this value to the discovery manager to resolve the server instance that provides this interface.
```

Comment:
There is no "discovery manager". Use "discovery service".

Reply:
if this gets a more official name, even "NI Discovery Service" we will need to remember to come back and update.

Reply:
The documentation currently calls it "the MeasurementLink™ discovery service".

Reply:
Thanks, I knew I was going to get some part of the terminology wrong.